framework/messages/hi/yii.php

```diff
@@ -0,0 +1,138 @@
+    'The image "{file}" is too small. The width cannot be smaller than {limit, number} {limit, plural, one{pixel} other{pixels}}.' => 'छवि "{file}" बहुत छोटी है। चौड़ाई {limit, number} {limit, plural, one{पिक्सेल} other{पिक्सेल}} से छोटी नहीं हो सकती।',
+    'The requested view "{name}" was not found.' => 'अनुरोधित दृश्य "{name}" नहीं मिला।',
+    'The verification code is incorrect.' => 'सत्यापन कोड गलत है।',
+    'Total <b>{count, number}</b> {count, plural, one{item} other{items}}.' => 'कुल <b>{count, number}</b> {count, plural, one{चीज} other{चीज़े}}।',
+    'Unable to verify your data submission.' => 'आपके डेटा सबमिशन को सत्यापित करने में असमर्थ।',
+    'Unknown alias: -{name}' => 'अज्ञात उपनाम: - {name}',
+    'Unknown option: --{name}' => 'अज्ञात विकल्प: - {name}',
+    'Update' => 'अद्यतन',
+    'View' => 'अवलोकन',
+    'Yes' => 'हाँ',
+    'Yii Framework' => 'Yii फ़्रेमवर्क',
+    'You are not allowed to perform this action.' => 'आपको यह कार्रवाई करने की अनुमति नहीं है।',
+    'You can upload at most {limit, number} {limit, plural, one{file} other{files}}.' => 'आप अधिकतम {limit, number} {limit, plural, one{फ़ाइल} other{फाइलें}} अपलोड कर सकते हैं।',
+    'in {delta, plural, =1{a day} other{# days}}' => '{delta, plural, =1{एक दिन} other{# दिनों}} में',
+    'in {delta, plural, =1{a minute} other{# minutes}}' => '{delta, plural, =1{एक मिनट} other{# मिनट}} में',
+    'in {delta, plural, =1{a month} other{# months}}' => '{delta, plural, =1{एक महीना} other{# महीने}} में',
+    'in {delta, plural, =1{a second} other{# seconds}}' => '{delta, plural, =1{एक सेकन्ड} other{# सेकन्डे}} में',
+    'in {delta, plural, =1{a year} other{# years}}' => '{delta, plural, =1{एक वर्ष} other{# वर्षों}} में',
+    'in {delta, plural, =1{an hour} other{# hours}}' => '{delta, plural, =1{एक घंटा} other{# घंटे}} में',
+    'just now' => 'अभी',
+    'the input value' => 'इनपुट मूल्य',
+    '{attribute} "{value}" has already been taken.' => '{attribute} "{value}" पहले से ही ले लिया गया है।',
+    '{attribute} cannot be blank.' => '{attribute} रिक्त नहीं हो सकती।',
+    '{attribute} contains wrong subnet mask.' => '{attribute} में गलत सबनेट मास्क है।',
+    '{attribute} is invalid.' => '{attribute} अमान्य है।',
+    '{attribute} is not a valid URL.' => '{attribute} एक मान्य URL नहीं है।',
+    '{attribute} is not a valid email address.' => '{attribute} एक मान्य ईमेल एड्रेस नहीं है।',
+    '{attribute} is not in the allowed range.' => '{attribute} अनुमत सीमा में नहीं है।',
+    '{attribute} must be "{requiredValue}".' => '{attribute} "{requiredValue}" होना चाहिए।',
+    '{attribute} must be a number.' => '{attribute} एक संख्या होनी चाहिए।',
+    '{attribute} must be a string.' => '{attribute} एक या ज़्यादा अक्षर होने चाहिए।',
+    '{attribute} must be a valid IP address.' => '{attribute} एक वैध IP एड्रेस होना चाहिए।',
+    '{attribute} must be an IP address with specified subnet.' => '{attribute} निर्दिष्ट सबनेट के साथ एक IP एड्रेस होना चाहिए।',
```

Comment:
'{attribute} must be an IP address with specified subnet.' => '{attribute} निर्दिष्ट सबनेट वाला IP एड्रेस होना चाहिए।',